aspnetcore/mvc/views/razor.md

```diff
@@ -60,6 +60,14 @@ With the exception of the C# `await` keyword, implicit expressions must not cont
 <p>@await DoSomething("hello", "world")</p>
 ```
 
+Implicit expressions may not contain C# generics, as the characters inside the brackets (`<>`) would be interpreted as an HTML tag:
```

Comment:
* may not :point_right: can't
* Perhaps link "C# generics" ... use ... \[C# generics](/dotnet/csharp/programming-guide/generics/)
* would be :point_right: are

Reply:
The generic link seems a little overkill, since there aren't any links to C# language features elsewhere in the topic, e.g., in the sub-sections on `using`, `try`/`catch`, and `lock`.  If we're assuming devs know about those features then they probably know about generics 😛

Reply:
> links to C# language features elsewhere in the topic

I'd put generics in a different, more advanced group of C# language features. There's no cost to the link; only upside. I think new devs might want to know more. I recommend providing it.

Reply:
Lol more advanced than `lock`?  You're right though, only downside I can think of is overwhelming the reader with a bunch of links, but there aren't that many others.

Reply:
imo the topics need *more* cross-links. Recently tho, everyone has been beefing up the links to API topics. I think that's going to pay off big time down the road. Generics is something that many devs have little or no contact with but a curiosity about. [@]BillWagner wrote a great topic on generics. There's no downside to linking anything that looks like something a new developer might have a curiosity about.

Reply:
> imo the topics need *more* cross-links.

As long as you can avoid link rot!  These new Microsoft Docs topics seem much better about that than the old MSDN articles though.  I submit to your good judgement on this.